### PR TITLE
add Arabic locale

### PR DESCRIPTION
**Pull request by fatimahossny**
Hi,
This to add Arabic Locale to the package.

Thank you for your efforts in this package :) .

### CODE REVIEW

File: lib/src/locale/arabic.dart [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+  String year(int amount, [bool abbreviated = true]) {
+    if (abbreviated) {
+      return 'ع';
+    } else {
+      if (amount == 1) {
+        return 'عام';
+      } else if (amount == 2) {
+        return 'عامين';
+      } else if (amount > 2 && amount < 11) {
+        return 'اعوام';
```
Comment (tekno0ryder) [inline lib/src/locale/arabic.dart:16]: أعوام, أسبوع, أشهر 
I think Is better

File: lib/src/locale/arabic.dart [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    }
+  }
+
+  @override
+  String millisecond(int amount, [bool abbreviated = true]) {
+    if (abbreviated) {
+      return 'م ث';
+    } else {
+      if (amount == 1) {
+        return 'ميلى ثانية';
```
Comment (tekno0ryder) [inline lib/src/locale/arabic.dart:139]: ملّي 
I think is better

**Review by tekno0ryder — COMMENTED** [review]
Thanks i really needed this, with minor comments below

File: lib/src/locale/arabic.dart (hunk 1 above)
Comment (fatimahossny) [reply lib/src/locale/arabic.dart]: Thanks , done :)

File: lib/src/locale/arabic.dart (hunk 2 above)
Comment (fatimahossny) [reply lib/src/locale/arabic.dart]: ok

**Comment by tejainece** [comment]
Thanks for the PR!

**Pull request merged**